it/controllers/internal/UpscanCallbackControllerISpec.scala

```diff
@@ -178,6 +212,26 @@ class UpscanCallbackControllerISpec extends IntegrationSpecCommonBase {
         duplicateModelInRepository.fileStatus shouldBe duplicateJsonAsModel.fileStatus
         duplicateModelInRepository.uploadDetails shouldBe duplicateJsonAsModel.uploadDetails
       }
+
+      "the body received is valid by the file has the same checksum as another file - duplicate different filename" in {
```

Comment:
Is this still needed?

Reply:
Added this test to cover the error same checksum, different filename case, but regardless of if the filename is the same or not, it'll still be classed as a duplicate so I'll remove it.